packages/opentelemetry-plugin-http/src/http.ts

```diff
@@ -311,10 +311,7 @@ export class HttpPlugin extends BasePlugin<Http> {
       };
 
       return context.with(propagation.extract(headers), () => {
-        const span = plugin._startHttpSpan(
-          `${method} ${pathname}`,
-          spanOptions
-        );
+        const span = plugin._startHttpSpan(`HTTP ${method}`, spanOptions);
```

Comment:
does it make sense to still add `pathname` to a span attribute or it will be rather a job for express plugin, @dyladan  ?

Reply:
It's already included in the `http.target` attribute

Reply:
@obecny was going to merge this but saw you haven't resolved this conversation. Just want to make sure you're ok with merging. Here is a link to the attribute I referred to: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-plugin-http/src/utils.ts#L347

Reply:
I have approved just after your explanation :), thx

Reply:
go ahead merge it pls